trace-protobuf/Cargo.toml

```diff
@@ -6,3 +6,11 @@ edition = "2021"
 
 [dependencies]
 prost = "0.11.6"
+serde = { version = "1.0.145", features = ["derive"] }
+
+[build-dependencies]
+prost-build = { version = "=0.11.3", optional = true  }
```

Comment:
explicitly use an older version of prost-build, as versions > 0.11.3 introduce adds `#[allow(clippy::derive_partial_eq_without_eq)]` to generated protobuf structs. This lint doesn't exist for rust versions < 1.63.0 and throws an `unknown lint` error in our CI

Reply:
it seems a bit dangerous to fix a older version for a new feature. are there any workarounds?

Reply:
A workaround would be to load and modify the autogenerated file in the build.rs rust script itself, removing the auto-added annotation that is causing problems.

I thought it would be cleaner not to do file processing in the code, but to just pin this earlier version. I couldn't figure out any other workarounds, what do you think the best approach would be?

Reply:
I've instead bumped the rust version we use in the CI tests `1.60.0` -> `1.63.0`

This lets us unpin the prost-build version, but if there are any objections I'll revert it.

Reply:
[this PR](https://github.com/DataDog/libdatadog/pull/77) will bump the rust version, I'll revert the changes here and wait that PR to merge.